sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/config/WebConfig.java

```diff
@@ -90,12 +101,35 @@ public void init(FilterConfig filterConfig) throws ServletException { }
             public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse,
```

Comment:
Can we abstract this filter as an individual authentication filter class?

Reply:
Yes, a new `AuthFilter` class has been added.